platform/platform.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	sshRetries = 120
+	sshRetries = 60
```

Comment:
Is this still enough for Equinix Metal? I think yes because it starts from the reboot of the PXE installer (there is a waiter first to check for the end of the PXE install) and I expect a single reboot to be done in less than 10 minutes.

Reply:
Im curious as well - I think it should be on the safe side, but I'm willing to increase it to 90 retries? The important thing is that we include the ssh timeout in the calculation, 100 minutes timeout for a test case where the whole suite takes <1hr is madness. We'll know more tomorrow after all tests have run.

Reply:
Let's leave it small then :)